templates/nem.mainnet.yaml

```diff
@@ -9,4 +9,4 @@ accounts:
   - { address: NA6U4U4Z5J3L5ZKSMZD5DKPWSVT5DASJNO4V5RAA, name: Trading1, roles: [trading] }
 
 nodes:
-  - host: 54.170.97.221
+  - host: xem43.allnodes.me
```

Comment:
would probably add multiple nodes for redundancy?

Reply:
yea, I was trying to think of how to pull available node at runtime but will add more nodes for now.